onnxruntime/test/providers/qnn/qnn_basic_test.cc

```diff
@@ -1098,6 +1099,38 @@ TEST_F(QnnHTPBackendTests, EPOffloadsGraphIOQuantDequant) {
   }
 }
 
+TEST_F(QnnHTPBackendTests, UseHtpSharedMemoryAllocatorForInputs) {
+#if !defined(__ANDROID__) && !defined(_WIN32)
```

Comment:
QC device for windows is Arm64 based, so you can check defined(__aarch64__) defined(_M_ARM64)